### PR TITLE
Ea 3180 onboard honey alice sbc msn usdp

### PR DESCRIPTION
**Pull request by evalenti-simtlix**
Ticket Number: EA-3180

### CODE REVIEW

**Review by Vijay-Jagannathan — DISMISSED** [review]
Approved but please provide a ticket number in the description

**Comment by evalenti-simtlix** [comment]
> Approved but please provide a ticket number in the description

Error founded, please re-review :)

**Review by noel-bitgo — APPROVED** [review]

File: modules/statics/src/coins.ts [hunk 1]
```diff
@@ -10040,6 +10056,38 @@ export const coins = CoinMap.fromCoins([
...
+    'bsc:xvs',
+    'Venus',
+    18,
+    '0xcf6bb5389c92bdda8a3747ddb454cb7a64626c63',
+    UnderlyingAsset['bsc:xvs']
+  ),
+  bscToken(
+    '8c59c402-b515-4238-9947-9240b5aed0b7',
+    'bsc:epx',
+    'Ellipsis',
```
Comment (mmcshinsky-bitgo) [inline modules/statics/src/coins.ts:10070]: Lists as `Ellipsis X` https://bscscan.com/address/0xaf41054c1487b0e5e2b9250c0332ecbce6ce9d71

File: modules/statics/src/coins.ts [hunk 2]
```diff
@@ -10971,6 +11019,30 @@ export const coins = CoinMap.fromCoins([
...
+    '13ff20df-fde8-478e-919b-eff8ea7169ab',
+    'txsgd',
+    'XSGD',
+    6,
+    '0x3f811bb6e605ef518b0cd9281eb4d9ad88a3953f',
+    UnderlyingAsset.TXSGD,
+    undefined,
+    undefined,
+    undefined,
+    Networks.test.goerli
```
Comment (mmcshinsky-bitgo) [inline modules/statics/src/coins.ts:11044]: This is holesky not goerli

**Pull request merged**